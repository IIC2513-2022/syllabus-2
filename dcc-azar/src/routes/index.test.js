const supertest = require('supertest');
const app = require('../app');

const request = supertest(app.callback());

describe('GET /', () => {
  test('responds correctly', async () => {
    const response = await request.get('/');
    expect(response.status).toBe(200);
    expect(response.text).toMatchSnapshot();
  });
});
