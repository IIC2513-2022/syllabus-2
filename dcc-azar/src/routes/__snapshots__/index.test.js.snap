// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET / responds correctly 1`] = `
"<!DOCTYPE html>
<html>

  <head>
    <meta charset=\\"utf-8\\">
    <title>IIC2513 - CHANGE-ME</title>
    
      <link rel=\\"stylesheet\\" href=\\"/assets/app.css\\">
    
  </head>

  <body>
    <header>
      <div id=\\"logo\\">
        <img src=\\"/assets/logo.png\\" alt=\\"Logo\\" />
      </div>
    </header>
    <h1>Hello world</h1>
<p>
  App web version
  1.0.0
</p>
<div id=\\"react-app\\">Here a React app will be rendered</div>

    <script src=\\"/assets/app.js\\"></script>
  </body>

</html>
"
`;
